docs/platform/release-notes/tizen-6-0-m1.md

```diff
@@ -0,0 +1,692 @@
+    - Generic web view: Categorized EWK API set systemically.
+    - .NET web view API
+      - Xamarin web view: Improves API coverage compared to the other platforms (iOS, Android).
+      - tizenFX Tizen.WebView: Improves API coverage compared to the Native web view.
+    - NUI Web view API: Provides API consistency (coverage, naming) with .NET webview.
+  - WRT-AddOn feature has been extended as follows:
+    - Handle multiple events from an addon.
+    - Handle multiple operations for an event.
+
+
+## Lightweight Web Solution
+
+### New and changed features
+
+- Rendering engine 
+  - WebRTC has been added.
+- JS Engine (New ES Features)
+  - Exponential operator has been added.
+  - New built-in methods (related with Array, Object) have been added.
+  - Rest parameter and spread operator for object have been added.
+- Performance and Memory Optimization
+  - Rendering Engine
+    - Direct mode has been enabled on EvasGL.
+    - CompositorGL has been fixed to enhance performance.
+    - Multithreaded image decoding feature has been added.
+    - OpenGL texture has been re-used to reduce the loading time.
+  - JS Engine
+    - Frequently used case about apply operation has been optimized.
+    - Source code compression has been updated to reduce large memory consumption in source code.
+
+### Fixes
+
+- Bug fixes
+  - Layout bugs have been fixed.
+  - Memory leak has been fixed.
+  - Crash has been fixed.
+
+
+## Tizen .NET
+
+### New and changed features
+
+- TizenFX (Tizen .NET Device API) API Level 8
+  - Pipeline and SingleShot functionalities have been added in MachineLearning Inference.
+  - Auto rotation sensor has been added in Sensor.
+  - LottieAnimationView has been added in NUI.
+- Xamarin Forms
+  - Xamarin.Forms 4.5.0, which is the latest stable version has been supported.
+- Tizen.CircularUI
+  - Wearable UI extension (Tizen.Wearable.CircularUI) 1.5.0 preview has been supported.
+- .NET runtime and tools
+  - .NETCore has been upgraded from 3.0.0 to 3.1.0.
+  - .NETCore diagnostics tools have been supported.
+
+
+## Toolchain
+
+### New and changed features
+
+- GNU GCC 9.2 has been enabled by default from Tizen 6.0 Platform.
+  - Libraries are updated by gmp 6.1.2, mpc 1.1.0, mpfr 4.0.2, and isl 0.22.
+  - Cilk runtime is dropped in upstream.
+- Glibc (GNU 2.30), Binutils (GNU 2.33.1) have been upgraded.
+  - The NIS (+) support library, libnsl, has been also deprecated from glibc 2.26.
+  - Libio.h has been removed from glibc 2.28.
+- Compilation options have been updated.
+  - The default mode for C++ is now -std=gnu++14 instead of -std=gnu++11.
+  - Program Instrumentation Option (-fstack-protector-strong) and Linker option (-Wl,-z,relro) have been added globally to improve security.
+  - Code generation Option (-frecord-gcc-switches) has been added and now used command line is recorded into the object file as .GCC.command.line.
+  - "_FILE_OFFSET_BITS=64" has been added as default on armv7l.

```

Comment:
Please remove the "":

 - _FILE_OFFSET_BITS=64 has been added as default on armv7l.

Suggest to use bold tag if you want to emphasize.

Reply:
will accept